packages/xo-web/src/common/xo/export-config-modal/index.js

```diff
@@ -0,0 +1,26 @@
+import _ from 'intl'
+import Component from 'base-component'
+import React from 'react'
+import { Password } from 'form'
+
+export default class ExportConfigModalBody extends Component {
+  get value() {
+    const { passphrase } = this.state
+    return passphrase === '' ? undefined : passphrase
```

Comment:
Trim before testing.

Reply:
Why?

Reply:
To delete empty spaces let by the user.

Reply:
But the passphrase could be `' '`, I don't think that's a problem, do you?

Reply:
Empty spaces can also be a user error.  I'll resolve this comment if you think it is necessary to let the possibility of the `' '` usage.

Reply:
It could be a mistake but just like any other typo. I don't think we should assume that the user doesn't want to set a passphrase containing only spaces.